exasol_sagemaker_extension/autopilot_utils/model_prediction.py

```diff
@@ -0,0 +1,21 @@
+import pandas as pd
+from sagemaker.deserializers import CSVDeserializer
+from sagemaker.predictor import Predictor
+from sagemaker.serializers import CSVSerializer
+
+
+class AutopilotPrediction:
+    """
+    This class is responsible for making prediction from a given Autopilot job
+    """
+
+    @staticmethod
+    def predict(endpoint_name: str, df: pd.DataFrame) -> list:
+        predictor = Predictor(endpoint_name)
+
+        predictor.serializer = CSVSerializer()
```

Comment:
could you check if other serializer are possible, for example https://sagemaker.readthedocs.io/en/stable/api/inference/serializers.html#sagemaker.serializers.NumpySerializer

Reply:
and if not please add a comment, that CSVSerializer and Deserializer are the only possible

Reply:
I tried Numpy Serialize but couldn't get it worked. It might be better to open a ticket to investigate this more?

Reply:
agree, lets investigate this later, then we stay for the moment with CSV